src/css/sections/reviews.css

```diff
@@ -0,0 +1,96 @@
+.review-cards-list {
```

Comment:
in general, we are using 2 spaces indents. Please, format the file

Reply:
Missed default formatter 🤦